provider/packet/packet_discover.go

```diff
@@ -23,11 +24,13 @@ func (p *Provider) SetUserAgent(s string) {
 // Help function
 func (p *Provider) Help() string {
 	return `Packet:
-	provider:		"packet"
-	project: 	 	UUID of packet project. Required
-	auth_token:   	Packet authentication token. Required
-	url: 		 	Packet REST URL. Optional
+	provider:       "packet"
+	project:        UUID of packet project. Required
+	auth_token:     Packet authentication token. Required
+	url:            Packet REST URL. Optional
 	address_type:   "private_v4", "public_v4" or "public_v6". Defaults to "private_v4". Optional
+	facility:       Filter for specific facility (Examples: "ewr1,ams1" (include),  "!ewr1,!nrt1" (exclude))
```

Comment:
I think we should remove the exclusion logic here. The reason being is that people can select a new tag to add to just nodes/hosts they want to bootstrap, and the exclusion logic adds some reasoning overhead (and more complexity in the implementation). I think just the "include" is the right thing here.

Reply:
I added the exclusion logic as I needed it for cross-site discovery. For example, I want the local consul cluster to connect to all of my clusters on the `wan` interface (I am currently doing that as external script but would love if `go-discover` could do that for me).
Can you suggest a way doing that without the exclude option?

Reply:
If I understand the use-case properly I would just explicitly configure which regions to join as well as which tags to join. If you were joining region `a` `b` `c` those would all be specified. If Packet added a new region `d`, you'd have to explicitly update that list.

I think that is more explicit and therefore a bit safer / more obvious. Same thing applies to hosts and tags too. Does that make sense?

Reply:
Okay, yeah, I was trying to avoid typing in all sites (we are going to have consul servers in each of Packet's locations) and to avoid making an action every time a new site is coming up.

It might be enough to just state 3-4 of Packet's main facilities and each of the smaller ones will try to connect to those main facilities and that should do the trick.

I'll update the PR.
Thanks!